R/oi_bicycle_parking.R

```diff
@@ -0,0 +1,37 @@
+#' Identify for bicycle parking.
```

Comment:
I suggest:

```
#' Identify bicycle parking places from OSM
```

There was a slight typo in the original and no need for a full stop.